src/pages/home/home.scss

```diff
@@ -1,139 +1,142 @@
 //Global
 
 h2 {
-    font-size: 2.85em;
-    font-family: "Poppins", sans-serif;
-    text-transform: uppercase;
-    color: #a0a0aa;
-    margin-bottom: 3rem;
-    @media (max-width: 766px) {
-        font-size: 2.5em;
-    }
+	font-size: 2.85em;
+	font-family: "Poppins", sans-serif;
+	text-transform: uppercase;
+	color: #a0a0aa;
+	margin-bottom: 3rem;
+	@media (max-width: 766px) {
+		font-size: 2.5em;
+	}
 }
 
 h3 {
-    font-family: "Poppins", sans-serif;
-    font-weight:100;
-    font-size: 1.5em;
-    text-transform: uppercase;
-    color: #a0a0aa;
-    margin-bottom: 3rem;
+	font-family: "Poppins", sans-serif;
+	font-weight: 100;
+	font-size: 1.5em;
+	text-transform: uppercase;
+	color: #a0a0aa;
+	margin-bottom: 3rem;
 }
 
 p {
-    font-family: "Lato", sans-serif;
-    font-size: 1.2em;
-    line-height: 1.5;
-    color: #a0a0aa;
+	font-family: "Lato", sans-serif;
+	font-size: 1.2em;
+	line-height: 1.5;
+	color: #ffffff;
```

Comment:
Since this is a generic p-tag, making this change impacts every p-tag on the home page. There are some paragraph elements that are not supposed to be white. You need to target the paragraph in the horizon section only for this ticket. This should be returned to the original #a0a0aa